DirectProgramming/DPC++FPGA/Tutorials/Features/loop_initiation_interval/README.md

```diff
@@ -0,0 +1,264 @@
+     ```
+     nmake fpga_emu
+     ```
+   * Generate the optimization reports: 
+     ```
+     nmake report
+     ``` 
+   * An FPGA hardware target is not provided on Windows*. 
+
+>**Note:** The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® PAC D5005 (with Intel Stratix® 10 SX FPGA) do not yet support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.
+
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Visual Studio* IDE (in Windows*). For instructions, refer to the following link: [Intel® oneAPI DPC++ FPGA Workflows on Third-Party IDEs](https://software.intel.com/en-us/articles/intel-oneapi-dpcpp-fpga-workflow-on-ide)
+
+## Examining the Reports
+
+Locate the pair of `report.html` files in either:
+
+* **Report-only compile**:  `loop_ii_report.prj` and `loop_ii_enable_ii_report.prj`
+* **FPGA hardware compile**: `loop_ii.prj` and `loop_ii_enable_ii.prj`
+
+Open the reports in any of Chrome*, Firefox*, Edge*, or Internet Explorer*. Looking at the reports for the design without the `intel::initiation_interval` attribute, navigate to the *Loop Analysis* report (*Throughput Analysis* > *Loop Analysis*). Click on the *SimpleMath* kernel in the *Loop List* panel and using the *Bottlenecks* viewer panel in the bottom left, you will see that a throughput bottleneck exists in the *SimpleMath* kernel. 
+
+Select the bottleneck. The report shows that the estimated f<sub>MAX</sub> is significantly lower than the target f<sub>MAX</sub> and shows the feedback path responsible, which is the feedback path in the initialization loop.
+
+The *Loop Analysis* report shows that the long-running loop achieves the target f<sub>MAX</sub> with an II of 1.
+
+Compare the results to the report for the version of the design using the `intel::initiation_interval` attribute. Here both loops achieve the target f<sub>MAX</sub>.
+
+>**Note:** Only the report generated after the FPGA hardware compile will reflect the true performance benefit of using the `initiation_interval` extension. The difference is **not** apparent in the reports generated by `make report` because a design's f<sub>MAX</sub> cannot be predicted. The final achieved f<sub>MAX</sub> can be found in `loop_ii.prj/reports/report.html` and `loop_ii_enable_ii.prj/reports/report.html` (after `make fpga` completes), in *Clock Frequency Summary* on the main page of the report.
+
+## Running the Sample
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+
+   ```bash
+   ./loop_ii.fpga_emu    # Linux
+   loop_ii.fpga_emu.exe  # Windows
+   ```
+
+2. Run the sample on the FPGA device (Linux only)
+
+   ```bash
+   ./loop_ii.fpga            # Sample without intel::initiation_interval attribute
+   ./loop_ii_enable_ii.fpga  # Sample with intel::initiation_interval attribute
```

Comment:
loop_ii?  enable_ii?

Reply:
This is an compiled/output file, so the loop_ii abbreviation is correct.